src/main/java/hudson/plugins/git/GitChangeSet.java

```diff
@@ -356,7 +356,8 @@ String getParentCommit() {
      * @param createAccountBasedOnEmail true if create new user based on committer's email.
      * @return {@link User}
      */
-    public User findOrCreateUser(String csAuthor, String csAuthorEmail, boolean createAccountBasedOnEmail) {
+    public User findOrCreateUser(String csAuthor, String csAuthorEmail, boolean createAccountBasedOnEmail,
```

Comment:
This removes the existing public API.  That's not allowed.  

The plugin doesn't know who is using the current public API among all the existing Jenkins plugins.  It's OK to add more API end points, but not to delete an API endpoint.